doc/README.md

```diff
@@ -191,33 +194,34 @@ paths with the highest cum weight.
   browser to view it.
 * **-png, -jpg, -gif, -pdf:** Generates a report in these formats.
 
-### Interpreting the Callgraph
+### Interpreting the Nodes
+
+![callgraph1](images/callgraph_nodes.svg)
 
-*   **Total Value**: amount of resource used by location and all location it
-    calls.
+*  _cum_ value is indiated by the nodes' color.
```

Comment:
Nit: Since we're planning to link from other places which don't use cum/self, it would be good to define self/cum here (even though it's redundant).

i.e. "cum value, the value of a location plus all of its descendants, is indicated"...

Reply:
Oops, sorry. I read your previous comment wrong.